RadixWallet/Features/DappInteractionFeature/Coordinator/DappInteractionModels.swift

```diff
@@ -131,6 +134,9 @@ extension DappToWalletInteraction {
 			// transactions
 			case .send:
 				0
+			// preAuthorization
```

Comment:
I would probably rename `send` to `submitTransaction` or similar. We now have `subintent` and then having just `send` looks weird. `send` was short for `sendTransasction` - and `sendTransaction` I've always disliked, I think it should be called `submitTransaction` and then we can call `subintent` `signSubintent` perhaps - because we actually do NOT submit the signed subintent, the Dapp does.

Reply:
100% agree (always disliked `send` as well). Updating